modules/core/src/main/java/io/fluo/recipes/transaction/RecordingTransaction.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2015 Fluo authors (see AUTHORS)
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.fluo.recipes.transaction;
+
+import io.fluo.api.client.Transaction;
+import io.fluo.api.exceptions.CommitException;
+
+public class RecordingTransaction extends RecordingTransactionBase implements Transaction {
+
+  private final Transaction tx;
+
+  protected RecordingTransaction(Transaction tx) {
+    super(tx);
+    this.tx = tx;
+  }
+
+  @Override
+  public void commit() throws CommitException {
+    tx.commit();
+  }
+
+  @Override
+  public void close() {
+    tx.commit();
```

Comment:
should call `tx.close()` not `tx.commit()`  I suppose this it where unit test w/ EasyMock would be good.. could verify that tx.close was called on a mock transaction